app/src/androidTest/java/org/projectbuendia/client/ui/chart/PatientChartActivityTest.java

```diff
@@ -261,17 +216,33 @@ public void testEncounter_latestEncounterIsAlwaysShown() {
         for (int i = 0; i < 6; i++) {
             openEncounterForm();
 
-            String pulse = Integer.toString(i + 80);
-            String temp = Integer.toString(i + 35) + ".0";
-            String vomiting = Integer.toString(5 - i);
-            answerTextQuestion("Pulse", pulse);
+            //String pulse = Integer.toString(i + 80);
+            //String temp = Integer.toString(i + 35) + ".0"; TODO: no decimal values on the chart. Is it a bug?
+            String temp = Integer.toString(i + 35);
+            String respiratoryRate = Integer.toString(i + 80);
+            String bpSystolic = Integer.toString(i + 80);
+            String bpDiastolic = Integer.toString(5 + 100);
```

Comment:
the bpDiastolic is being the same for each interaction. It should be "i" instead of "5"